front50-gcs/src/main/java/com/netflix/spinnaker/front50/config/GcsConfig.java

```diff
@@ -53,6 +54,18 @@
 @ConditionalOnExpression("${spinnaker.gcs.enabled:false}")
 @EnableConfigurationProperties(GcsProperties.class)
 public class GcsConfig {
+  @Value("${spinnaker.gcs.safeRetryMaxWaitIntervalMs:60000}")
+  Long maxWaitInterval;
+
+  @Value("${spinnaker.gcs.safeRetryRetryIntervalBaseSec:2}")
```

Comment:
Why not put all of these under a `safeRetry` block? `safeRetryRetry...` reads kind of funny